components/brave_rewards/resources/ui/components/adsBox.tsx

```diff
@@ -140,39 +286,58 @@ class AdsBox extends React.Component<Props, State> {
     const toggle = !(!enabledMain || !adsUIEnabled || !adsIsSupported)
     const showDisabled = firstLoad !== false || !toggle || !adsEnabled || !adsIsSupported
 
+    if (!adsHistory) {
```

Comment:
we still need to show ads box even if history doesn't exist

Reply:
Do we? I thought the idea was to enable/disable the link for it based on history/no-history (that's currently how it works).

Reply:
cc @jenn-rhim My understanding is that Ads history link is not shown if there isn't any history.

Reply:
correct, if there are no history link shouldn't be displayed

Reply:
Hi @jenn-rhim, any thoughts on this one? We always show the Ads History link at the moment, but we disable it if there are no entries. Could you confirm if that's how this should behave? Thanks!

Reply:
@emerick My opinion is that we either have the link in a disabled format, or not show it at all when there's no history.  No need to tempt users to go visit the page where no content is available. For learnability perspective, I'd advocate having it in a disabled format. The odds are that the user's having this state of not seeing any in the history would be a very, very short period of time once enabled the ads. @NejcZdovc

Reply:
OK, sounds good!

Reply:
I would just hide the link, the same way we do it in AC box

Reply:
OK, I'll get that in there then.

Reply:
Fixed.